README.md

```diff
@@ -95,11 +89,4 @@ Common issues which people face are:
 
 
 ## © License
-This project is licensed under the alina101, e-lemma, Lasped13, Zhi-704 - see the LICENSE.md file for details.
-
-
-## ❤️ Acknowledgements
-- 🎹 **Bandcamp** at the LMNH for taking care of the plants.
-- 🧡 **Sigma Labs** for giving us this project.
-- 🤖 **Sigma Bot** for helping us with the project.
-- 🎻 **Team Apollo** for creating this.
\ No newline at end of file
+This project is licensed under the alina101, e-lemma, Lasped13, Zhi-704 - see the LICENSE.md file for details.
```

Comment:
What does this line mean?

Reply:
Ah, I left it in because even I was unsure what this meant. This was in the previous readme but I might just delete it now.